src/reason_pprint_ast.ml

```diff
@@ -3014,6 +3011,23 @@ class printer  ()= object(self:'self)
         in
         let rightItm = self#simplifyUnparseExpr rightExpr in
         Some (label ~space:forceSpace (atom prefixStr) rightItm)
+      | (Infix s, [(_, e1); (_, e2)]) when s.[0] = '#' ->
+        (*
+          If the expression parsed is
+            expr1##expr2
+          then keeping no spaces between that infix operator and the expressions
+          is important as ## signifies "js property access" and therefore looks
```

Comment:
I think it signifies "extern access" - it could be interoperating with an C++ object etc.